src/block/edit.tsx

```diff
@@ -60,16 +60,19 @@ export default function Edit( { attributes, setAttributes, clientId }: Props ) {
 
 	const blockProps = useBlockProps();
 
-	const authorEmail = useSelect( ( select: SelectFn ) => {
-		const postType = select( 'core/editor' ).getCurrentPostType();
-		const postId = select( 'core/editor' ).getCurrentPostId();
-		const authorId = select( 'core' ).getEntityRecord( 'postType', postType, postId )?.author;
+	// Get author email from postType and postId in the context.
+	const authorEmail = useSelect(
+		( select: SelectFn ) => {
+			const { getEditedEntityRecord, getEntityRecord } = select( 'core' );
+			const _authorId = getEditedEntityRecord( 'postType', postType, postId )?.author;
```

Comment:
Does `_authorId` signify a private variable? We don't use it in the profile block. Moreover, `_authorId` is available only within the function's scope, so I recommend using `authorId` directly.

> Note: Re the `_debounce`, which represents [the lodash debounce](https://www.lodashjs.com/docs/lodash.debounce/#_debouncefunc-wait0-options)